project/Testing.scala

```diff
@@ -8,31 +8,44 @@ object Testing {
+  lazy val UseCromwellIntegrationTaggedTests = Tests.Argument("-n", CromwellIntegrationTestTag)
+  lazy val DontUseCromwellIntegrationTaggedTests = Tests.Argument("-l", CromwellIntegrationTestTag)
+
+  lazy val DbmsTestTag = "DbmsTest"
+  lazy val UseDbmsTaggedTests = Tests.Argument("-n", DbmsTestTag)
+  lazy val DontUseDbmsTaggedTests = Tests.Argument("-l", DbmsTestTag)
 
   /*
   The arguments that will be added to the default test config, but removed from all other configs.
   `sbt coverage test` adds other arguments added to generate the coverage reports.
   Tracking the arguments we add to the default allows one to later remove them when building up other configurations.
  */
-  lazy val defaultTestArgs = Seq(Tests.Argument("-l", "DockerTest"), Tests.Argument("-l", "CromwellIntegrationTest"))
+  lazy val defaultTestArgs = Seq(DontUseDockerTaggedTests, DontUseCromwellIntegrationTaggedTests, DontUseDbmsTaggedTests)
 
   val testSettings = List(
-    // `test` (or `assembly`) - Run all tests, except docker and integration
+    // `test` (or `assembly`) - Run all tests, except docker and integration and DBMS
     testOptions in Test ++= defaultTestArgs,
     // `alltests:test` - Run all tests
     testOptions in AllTests := (testOptions in Test).value.diff(defaultTestArgs),
-    // `docker:test` - Run docker tests, except integration
-    testOptions in DockerTest := (testOptions in Test).value.diff(defaultTestArgs) ++
-      Seq(Tests.Argument("-n", "DockerTest"), Tests.Argument("-l", "CromwellIntegrationTest")),
+    // `notests:test` - Run no tests
+    testOptions in NoTests := (testOptions in Test).value ++ Seq(Tests.Filter(_ => false)),
+    // `docker:test` - Run only the docker tests.
+    testOptions in DockerTest := (testOptions in AllTests).value ++ Seq(UseDockerTaggedTests),
     // `nodocker:test` - Run all tests, except docker
-    testOptions in NoDockerTest := (testOptions in Test).value.diff(defaultTestArgs) ++
-      Seq(Tests.Argument("-l", "DockerTest")),
-    // `integration:test` - Run integration tests, except docker
-    testOptions in CromwellIntegrationTest := (testOptions in Test).value.diff(defaultTestArgs) ++
-      Seq(Tests.Argument("-l", "DockerTest"), Tests.Argument("-n", "CromwellIntegrationTest")),
+    testOptions in NoDockerTest := (testOptions in AllTests).value ++ Seq(DontUseDockerTaggedTests),
+    // `integration:test` - Run only integration tests
+    testOptions in CromwellIntegrationTest := (testOptions in AllTests).value ++ Seq(UseCromwellIntegrationTaggedTests),
     // `nointegration:test` - Run all tests, except integration
-    testOptions in CromwellNoIntegrationTest := (testOptions in Test).value.diff(defaultTestArgs) ++
-      Seq(Tests.Argument("-l", "CromwellIntegrationTest"))
+    testOptions in CromwellNoIntegrationTest := (testOptions in AllTests).value ++ Seq(DontUseCromwellIntegrationTaggedTests),
+    // `dbms:test` - Run database management tests.
```

Comment:
Something is confusing me here... I tried running `sbt test` and `sbt dbms:test` and it appears that it runs the same number of tests (263)